GetStream/stream-chat-swift
# Use ContainerView in ChatChannelListItemView

# What this PR do:
- Uses `ContainerView` inside `ChatChannelListItemView` instead of constrained layout. This gives us more flexibility in the code and also fixe some minor issues with the layout in clone applications.
# How to test it
- Run test applications, see if they look OK, see if CI doesn't fail and mostly check the DEMO application. This replacement is actually fixing issue with `unreadCountView` being wrongly aligned when there is no title to the channel.-
